edit.html

```diff
@@ -33,10 +34,15 @@
 
     <script src="vendor/codemirror/addon/edit/matchbrackets.js"></script>
 
-    <link rel="stylesheet" href="vendor/codemirror/addon/lint/lint.css" />
+    <!-- CSSLint -->
```

Comment:
* genericDisabledLabel is okay, I forgot about it.
* ideally, stylelint should be the default, but it's too soon to make it so, we and other users need to test it, you may be able to reduce the bundle etc.
* so, practically, we should preserve the historic behavior: keep csslint